src/OpenLoco/src/Input/Shortcuts.cpp

```diff
@@ -397,7 +409,30 @@ namespace OpenLoco::Input::Shortcuts
     // 0x004BF3DC
     static void sendMessage()
     {
-        call(0x004BF3DC);
+        if (isEditorMode())
+            return;
+
+        if (!isNetworked())
+            return;
+
+        if (isTitleMode())
+        {
+            auto* caller = WindowManager::find(WindowType::titleMenu);
+            if (caller == nullptr)
+                return;
+
+            const WidgetIndex_t callingWidget = 4;
+            Windows::TitleMenu::beginSendChatMessage(caller, callingWidget);
```

Comment:
The widgetindex can just be internal for the titlemenu (they're referring to the same widget. But keeping it inside the window doesn't leak the widx)